<launch>
  <node pkg="rosbag" type="play" name="rosbag" required="true" args="/home/stevenliu/bagfiles/day2.bag"/>
  <node name="extract" pkg="rowbot_vision" type="bagToVid.py" cwd="ROS_HOME">
    <param name="channel" value="front_camera/image_raw"/>
  </node>
</launch>